@navikt/core/react/src/form/combobox/FilteredOptions/filteredOptionsContext.tsx

```diff
@@ -81,19 +68,32 @@ export const FilteredOptionsProvider = ({ children, value: props }) => {
       return externalFilteredOptions;
     }
     const opts = [...customOptions, ...options];
-    setFilteredOptionsIndex(null);
-    return getMatchingValuesFromList(searchTerm, opts);
+    return filteredOptionsUtils.getMatchingValuesFromList(searchTerm, opts);
   }, [customOptions, externalFilteredOptions, options, searchTerm]);
 
   const previousSearchTerm = usePrevious(searchTerm);
 
   const [isMouseLastUsedInputDevice, setIsMouseLastUsedInputDevice] =
     useState(false);
 
+  const filteredOptionsMap = useMemo(
+    () =>
+      options.reduce(
+        (map: Map<string, string>, _option: string) => ({
```

Comment:
Feil type på `map`. Hvis du definerer typer på `FilteredOptionsProvider` sine props, så slipper du å definere noen typer her.

Reply:
Helt korrekt. 👍  Definerte typer for props, og gjenbrukte fra ComboboxProps, så oppdaget jeg litt feiltyping rundt om.